QUARKUS-2991.md

```diff
@@ -0,0 +1,128 @@
+# QUARKUS-2991 Support for MicroProfile 6
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-2991
+
+Quarkus documentation:
+- https://quarkus.io/version/main/guides/opentelemetry
+- https://quarkus.io/version/main/guides/openapi-swaggerui
+- https://quarkus.io/version/main/guides/security-jwt
+- https://quarkus.io/version/main/guides/smallrye-metrics
+
+
+## Scope of testing
+
+MicroProfile 6.0 brings upgrades a few of its standards.
+
+RHBQ 3.x aims to support the following set of MicroProfile specifications:
+- Telemetry 1.0 - new in MP 6
+- Open API 3.1 - updated in MP 6
+- Rest Client 3.0
+- Config 3.0
+- Fault Tolerance 4.0
+- JWT Authentication  2.1 - updated in MP 6
+- Health 4.0
+- Jakarta EE 10 Core Profile - updated in MP 6
+
+RHBQ 3.x should also support:
+- Reactive Messaging 3.0 - updated in MP 6
+- Reactive Streams Operators 3.0 - updated in MP 6
+- Context Propagation 1.3
```

Comment:
Updated in MP 5 which Quarkus intentionally skipped and stayed on MP 4.1 specs

Reply:
Mentioned further in the TP.